test/system/jsdoc-config.test.js

```diff
@@ -18,16 +17,16 @@ describe('JSDoc configuration', function () {
     });
 
     it('should have readable content', function () {
-        expect(content = fs.readFileSync(jsdocConfigPath).toString()).to.be.ok();
+        expect(content = fs.readFileSync(jsdocConfigPath).toString()).to.be.ok;
```

Comment:
Add proper assertion messages, so that failures can be traced easily for assertions like this.